data/human/transport missions.txt

```diff
@@ -3632,3 +3632,123 @@ mission "Antibiotics To South"
+				decline
+			label offer
+			`	"I happen to take an interest in young pioneering privateers, and your performance, even in a small vessel like the <ship>, has impressed me greatly. I'll tell you more if you let me take you somewhere."`
+			choice
+				`	"Sure, I'll follow."`
+					goto follow
+				`	"Where are you taking me?"`
+			`	The man chuckles nervously. "Just to somewhere more private. Don't worry, I won't bite."`
+			choice
+				`	"Fine, I'll go."`
+				`	"I still don't buy this. I'm out."`
+					decline
+			label follow
+			`	You follow the man into a series of winding alleyways, keeping your eyes alert for any danger. After a while, the man stops. "I suppose I can tell you more about myself now. I... am the leader of the local militia, and I believe that, considering your proficiency in piloting smaller craft, you would be a great candidate to join our team. However, there is a test you must pass. I will use one of our smaller ships in our fleet, the Esca, to challenge you to a duel in the system of Umbral. If you manage to disable me, go to our base in Wayfarer, and I'll let you join."`
+			choice
+				`	"What sort of vessel is the Esca?"`
+					goto vessel
+				`	"Why did you need to bring me to here? Couldn't you have made this offer in the spaceport?"`
+			`	The man hesiates for a moment. "Well, being in the open when admitting that you're the leader of the ragtag that hunts down pirates in a potentially pirate infested spaceport isn't a good idea. You can't ever be too sure about who's listening.`
+				goto end
+			label vessel
+			`	The man hesiates for a moment. "Oh, just some low-end Southbound Shipyards light warship. Nothing to worry about for a captain like you.`
+			label end
+			`	"Well, I better be going now. Meet me at Umbral as fast as you can if you want to take up my challenge." The man walks away, leaving you to your own devices.`
+			choice
+				`	(Ready your ship for the battle with the Esca.)`
+					accept
+				`	(Ignore the invitation.)`
+					decline
+	
+	on enter
+		dialog `When you take off, you run a quick scan on all other ships that are taking off. You see no sign of the Esca. Maybe it's already waiting for you at Umbral.`
+	on enter "Umbral"
+		"reputation: Independent (Friendly before duel)" = -1000
+		conversation
+			`When you enter Umbral, you continue to see no sign of the Esca. Suddenly, an alarm rings out as your sensors detect that a pirate Vanguard has entered the system along side you. Before you can respond, your ship intercepts a hail, and it opens communication.`
+			`The voice of the alleged captain of the Esca shouts out towards you. "Ho ho ho! Did you really think that I was the leader of the militia? Well, considering that you're here, I suppose you did. And now you'll pay for your mistake! Don't bother trying to call for help either, the militia don't patrol this system!"`
+			`The Vanguard powers up its weapons.`
+	
+	npc kill
+		government "Independent (Friendly before duel)"
```

Comment:
A dummy " Independent " government already exists for purposes like this. Use that government instead, and set the reputation to positive on accept in order to guarantee that the government is friendly. (And perhaps also reset it to friendly when the mission is complete as to not ruin other mission that use this government.)

Reply:
Alternatively, you could save the " Independent " government's reputation to a condition on accept, then reset the government's reputation to that condition on complete (instead of simply resetting the reputation to friendly).